apps/server/README.md

```
@@ -0,0 +1,70 @@
+## Development
+
+```bash
+git clone https://github.com/freedit-org/freedit
+cd freedit && cargo run
+```
+
+## Credits
+
+* icon: <https://iconoir.com/>
+* CSS framework: <https://bulma.io/>
+* Rust crates: [Cargo.toml](https://github.com/freedit-org/freedit/blob/main/Cargo.toml)
```

Markdownlint (0.37.0)

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)